README.md

```diff
@@ -132,6 +132,7 @@ The kernel has to be built with the following configuration:
 - `CONFIG_BPF_SYSCALL=y`
 - `CONFIG_DEBUG_INFO_BTF=y`
 - `CONFIG_FTRACE=y`
+- `CONFIG_KPROBES=y`
```

Comment:
As this isn't a requirement for sched_ext proper, can you create a separate section for clarity?